src/include/execution/vm/bytecode_handlers.h

```diff
@@ -1490,9 +1490,8 @@ VM_OP_WARM void OpCharLength(terrier::execution::sql::Integer *result, terrier::
 }
 
 VM_OP_WARM void OpConcat(terrier::execution::sql::StringVal *result, terrier::execution::exec::ExecutionContext *ctx,
-                         const terrier::execution::sql::StringVal *left,
-                         const terrier::execution::sql::StringVal *right) {
-  terrier::execution::sql::StringFunctions::Concat(result, ctx, *left, *right);
+                         const int64_t *num_inputs, const terrier::execution::sql::StringVal **inputs) {
```

Comment:
Other functions in this file that pass in arrays and such don't pass in the length of the array or in this case number of inputs as a pointer but the actual value. Is there a specific reason this is a pointer by any chance?

Reply:
No good reason, I'll de-reference it in vm.cpp so it's not a pointer here.

Reply:
Updated